src/main/kotlin/org/wycliffeassociates/sourceaudio/upload/RequestHandler.kt

```diff
@@ -35,23 +37,30 @@ class RequestHandler {
 
     private fun processUploadModel(file: File, params: Map<String, String>): String {
         val response = mutableMapOf<String, Any>()
+        var output: String = ""
+        var isSuccess: Boolean = true
+
         try {
             val model = createModel(file, params)
-            response.putAll(
-                mapOf(
-                    "output" to FilePathGenerator.createPathFromFile(model),
-                    "success" to true
-                )
-            )
-        } catch (ex: IllegalArgumentException) {
-            response.putAll(
-                mapOf(
-                    "output" to ex.message!!,
-                    "success" to false
-                )
-            )
+
+            output = FilePathGenerator.createPathFromFile(model)
+
+            val newDirectory = File("./src/main/resources/uploads/$output")
```

Comment:
./src/main/resources/uploads/ 
This will be replaced with server's root directory. We'll try to put it in a key-value in a config file